src/foam/u2/view/BadgeView.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * @license
+ * Copyright 2021 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+foam.CLASS({
+  package: 'foam.u2.view',
+  name: 'BadgeView',
+  extends: 'foam.u2.View',
+  
+  css:`
+    ^Wrapper{
+      height: 24px;
+     
+      width: 79px;
+
+      padding: 0 8px;
+ 
```

Comment:
You are taking over the css ... but was all this space intentional. I am under the impression that uniformity is easier to read.

Reply:
Nope, Will remove